### PR TITLE
feat(event): addParticipantOnEvent mutation

### PR DESCRIPTION
**Pull request by vinibgoulart**

### CODE REVIEW

File: apps/server-admin/src/modules/event/mutations/AddParticipantInEventMutation.ts [hunk 1]
```diff
@@ -45,19 +40,33 @@ const AddUserInEventMutation = mutationWithClientMutationId({
       return { id: null, success: null, error: t('Unauthorized') };
     }
 
+    // i don't like it
+    // https://dev.to/woovi/processing-promises-in-batch-2le6
```
Comment (vinibgoulart) [inline apps/server-admin/src/modules/event/mutations/AddParticipantInEventMutation.ts:44]: how to improve it?

File: apps/server-admin/src/modules/event/EventMutations.ts [hunk 2]
```diff
@@ -1,9 +1,9 @@
-import { AddUserInEventMutation } from './mutations/AddUserInEventMutation';
+import { AddParticipantInEventMutation } from './mutations/AddParticipantInEventMutation';
 import { CreateEventMutation } from './mutations/CreateEventMutation';
 import { UpdateEventMutation } from './mutations/UpdateEventMutation';
 
 export const EventMutations = {
   CreateEventMutation,
   UpdateEventMutation,
-  AddUserInEventMutation,
+  AddParticipantInEventMutation,
```
Comment (daniloab) [inline apps/server-admin/src/modules/event/EventMutations.ts:8]: prefer FeatureActionField

feature -> the feature being write
action -> the action add, edit, delete, remove, update
field -> when is a field being changed or the secondary action

EventAddParticipant
EventCreate
EventUpdate

also, what is the differente between add, create and update? looks like you are following REST approaches

Comment (vinibgoulart) [reply apps/server-admin/src/modules/event/EventMutations.ts:8]: AddParticipantInEvent is a mutation that create a participant and update an event

should I avoid it? and run them separately?

Comment (vinibgoulart) [reply apps/server-admin/src/modules/event/EventMutations.ts:8]: instead i run:
- CreateParticipant
- UpdateEvent

i just run 
- AddParticipantInEvent

**Pull request merged**